src/middleware/processJwt.ts

```diff
@@ -4,12 +4,10 @@ import type { NextFunction, Request, Response } from 'express';
 
 const expressJwtMiddleware = expressjwt(jwtOptions);
 
-export const verifyJwt = (
-	req: Request,
-	res: Response,
-	next: NextFunction,
-): void => {
+const processJwt = (req: Request, res: Response, next: NextFunction): void => {
 	expressJwtMiddleware(req, res, next).catch((error: unknown) => {
-		throw error;
+		next(error);
```

Comment:
We should make sure this is actually a correct change -- it might be, but maybe the correct response here is to log the error rather than invoke `next`.

It's unfortunate because it's not clear if expressJwtMiddleware will invoke next if it fails (if it does, then I think calling next twice would result in a problem; if it doesn't then I believe the request would hang).

Given that... maybe the safest approach would be to log it.  I worry that if next is called twice it could crash the process, but if next is called zero times the damage is just that single connection.

Reply:
(in either event I think throwing would crash the process, and so that's definitely not what we want)